documentation/upgradingToTeams.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: documentation
+menu: upgradingToTeams
+subtitle: Upgrading to Teams
+---
+
+# Upgrading to Teams
+
+This article assumes you are a proud owner of a Flyway Teams license. If you are not, head over to the [download/pricing page](/download) to purchase a license or start a free trial.
+
+## Upgrading from community
+
+If you are currently using Flyway Community there are a few things you need to do to upgrade to using Flyway Teams. The steps to do so differ depending on how you are accessing Flyway, but broadly consist of two main steps. Telling Flyway to use the [correct edition](/documentation/configuration/edition), and supplying a [license key](/documentation/configuration/licenseKey). See below for more details for each usage.
+
+### Command Line
+
+If you are using the command line, simply add the `-teams` flag when you invoke Flyway (or use the environment variable `FLYWAY_EDITION=teams`). Then provide your [license key](/documentation/configuration/licenseKey) through any of the supported methods.
+
```

Comment:
Maybe a little header above the code blocks that says `example` or adjust the previous sentence to say `any of the following supported methods.`